test/Parse/enum.swift

```diff
@@ -308,12 +308,12 @@ enum NonliteralRawValue : Int {
   case Yeon = 100 + 20 + 3 // expected-error {{raw value for enum case must be a literal}}
 }
 
-enum RawTypeWithPayload : Int { // expected-note {{declared raw type 'Int' here}} expected-note {{declared raw type 'Int' here}}
+enum RawTypeWithPayload : Int { // expected-error {{type 'RawTypeWithPayload' does not conform to protocol 'RawRepresentable'}} expected-note {{declared raw type 'Int' here}} expected-note {{declared raw type 'Int' here}}
   case Powell(Int) // expected-error {{enum with raw type cannot have cases with arguments}}
   case Terwilliger(Int) = 17 // expected-error {{enum with raw type cannot have cases with arguments}}
 }
 
-enum RawTypeMismatch : Int {
+enum RawTypeMismatch : Int { // expected-error {{type 'RawTypeMismatch' does not conform to protocol 'RawRepresentable'}}
   case Barbur = "foo" // expected-error {{}}
 }
 
```

Comment:
It could be that in the interpreter, we go ahead and type check the bodies all synthesized members, but in the test, we do not.

Remember that bodies of synthesized members are built lazily, and we only crashed because we'd hit a case where a synthesized member subsequently had no body.